README.md

```diff
@@ -4,6 +4,8 @@ Dink sends webhook messages upon noteworthy in-game events.
 While Dink supports the Discord webhook format (with rich embeds and optional screenshots), it also includes additional metadata that allows custom webhook servers to analyze messages or even generate their own messages.
 This project was forked from UniversalDiscordNotifier, but has more features, reliability, configurability, testing, and maintainer activity. 
 
+Have a suggestion, bug report, or question? Let us know at our [issue tracker](https://github.com/pajlads/DinkPlugin/issues)!
```

Comment:
```suggestion
Have a suggestion (e.g., new notifier, additional data), bug report (as rare as it may be), or question? Let us know at our [issue tracker](https://github.com/pajlads/DinkPlugin/issues)!
```

Reply:
I'm happy with the suggestion :+1: